src/index.js

```diff
@@ -1,6 +1,6 @@
 import { Server } from 'hapi';
 import CustomersPlugin from './plugins/customers';
-
+import Authentication from './plugins/authentication';
```

Comment:
Rename the plugin to standardize the plugins with a suffix of Plugin `AuthenticationPlugin`